snap/info.go

```diff
@@ -689,6 +691,7 @@ func (app *AppInfo) SecurityTag() string {
 	return AppSecurityTag(app.Snap.Name(), app.Name)
 }
 
+// DesktopFile returns the path to the installed optional desktop file for the application.
```

Comment:
`.. , optionally installed, application desktop file`?

Reply:
the original file is optional but if present is installed. This formulation is quirky but the other doesn't convey that to me

Reply:
How about
> ```
> DesktopFile returns the installed path to the optional desktop file for the application.
> ```
?